src/main/java/lotto/utils/StringAddCalculator.java

```diff
@@ -0,0 +1,76 @@
+package lotto.utils;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class StringAddCalculator {
+
+    private final static Pattern pattern = Pattern.compile("//(.)\n(.*)");
+    public final static String DEFAULT_SPLIT_REGEX = ",|:";
+
+    public void stringCalculator(String inputString) {
+        int sum = getNumberList(inputString);
+    }
+
+    public int getNumberList(String inputString) {
```

Comment:
해당 부분 잘 구현해주셨습니다. 소소한 코멘트를 드리자면

> 문자열을 전달하는 경우 구분자를 기준으로 분리한 각 숫자의 합을 반환
> 

`StringAddCalculator` 는 위의 요구사항을 만족하기 위해 아래 세가지 책임을 가지고 있습니다.

1. 구분자를 기준으로 문자열을 구분한다.
2. 문자열을 솟쟈로 변환한다.
3. 숫자의 합을 구한다.

구현해주신 코드에서 메서드 분리를 통해 충분히 가독성있고 좋아 보이지만,생각보다 많은 책임을 수행하고 있을 수 있으며 이에 대한 각각의 단위테스트는 수행하기 힘든 형태입니다.따라서 다음 로또 미션에서는 책임의 분배와 클래스를 작은 단위로 분리 하는 것에 대해 생각하고 구현해보신다면 앞으로의 미션에도 많은 도움이 될 것이라 생각이 됩니다 :)


추가적으로 기존의 메서드 명 에서는 숫자를 sum한다는 생각을 하지 못하고, 문자열을 List<Integer>로 변환하여 반환할 것이이다 라고 생각이 드는데요!  🤔  좀 더 적합한 메서드 명이 필요한 때인 것 같아요 :)
```suggestion
    public int splitAndSum(String inputString) {
```

Reply:
단위 테스트가 편한 상태로 코딩하는게 쉽지 않아서 고민이네요😭😭😭
근데 리뷰어님이 남겨주신 코멘트를 보고 머리가 조금 띵 했습니다!!!
클래스와 메소드의 책임을 바라보는 시선으로 코딩한다면 뭔가 개선될것 같습니다.🤔